twtr/src/integration-test/groovy/twtr/MessageFunctionalTests/CreateMessagesErrorsSpec.groovy

```diff
@@ -0,0 +1,175 @@
+package twtr.MessageFunctionalTests
+
+import grails.test.mixin.integration.Integration
+import groovyx.net.http.HttpResponseException
+import groovyx.net.http.RESTClient
+import spock.lang.*
+import geb.spock.*
+
+@Integration
+@Stepwise
```

Comment:
If you skip the first test and just add the account in the global setup() and do a delete in the global cleanup(), you don't need the ```@Stepwise``` tag.